src/utils/Theme/themes/dark.ts

```diff
@@ -368,7 +368,7 @@ export const darkTheme: Theme = {
       'avatar-initials-color': darkColors.white,
       'avatar-border': 'solid 1px var(--body-background)'
     },
-    spacer: {
+    stack: {
       'spacer-gap': spacings.md,
```

Comment:
We probably need to rename these too which makes this a breaking change - i think its fine since its so minor so maybe we don't even alias and just push on.